cabal-install/tests/IntegrationTests2.hs

```diff
@@ -118,6 +123,23 @@ main = do
           (tests config)
     )
 
+-- Tests are run silently, unless they fail. Firstly because it is annoying to
+-- see lots of stderr from your unit tests. Secondly becaus this output
+-- leaks into the result of github actions (#8419)
+silentTest :: TestTree -> TestTree
+silentTest = wrapTest silentHelper
+  where
+    silentHelper t = do
+      (out, res) <- hCapture [stderr] t
```

Comment:
Could this capture `stderr` from other tests executed concurrently? Not a big deal obviously, but might be worth to add a comment or enforce single-threaded execution (if not enforced already).

Reply:
A good point, I tested with `-j8` without this patch and some tests fail anyway, so I will force this testsuite to run sequentially.